### PR TITLE
Add myself to Central Dashboard owners

### PR DESCRIPTION
**Pull request by avdaredevil**
Added myself to OWNERS so I can merge after LGTMs

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/kubeflow/2502)

### CODE REVIEW

File: components/centraldashboard/OWNERS [hunk 1]
```diff
@@ -1,3 +1,4 @@
 approvers:
   - swiftdiaries
+  - avdaredevil
```
Comment (jlewi) [inline components/centraldashboard/OWNERS]: alphabetize please

Comment (avdaredevil) [reply components/centraldashboard/OWNERS]: Done!

**Pull request merged**